internal/config/wrapper.go

```diff
@@ -230,6 +230,7 @@ func (w *Wrapper) InvalidateFolder(id string, err string) {
 	for i := range w.cfg.Folders {
 		if w.cfg.Folders[i].ID == id {
 			w.cfg.Folders[i].Invalid = err
+			events.Default.Log(events.ConfigSaved, w.cfg)
```

Comment:
I've been thinking now and then that "Invalid" should be a folder state like scanning or idle... Then it would be a normal state change.